apps/testingaccessibility/src/utils/format-prices-for-product.ts

```diff
@@ -286,6 +292,6 @@ async function couponForType(
   return merchantCoupons.map((coupon: any) => {
     // for pricing we don't need the identifier so strip it here
     const {identifier, ...rest} = coupon
-    return rest
+    return {...rest, ...(country && {country})}
```

Comment:
adding `country` here so that it's part of coupon object and can be used within the ppp banner